ceci/sites/cori.py

```diff
@@ -61,27 +65,27 @@ def command(self, cmd, sec):
             paths_end = "'" if paths else ""
             return (
                 f"{mpi1} "
-                f"shifter "
+                "shifter "
                 f"--env OMP_NUM_THREADS={sec.threads_per_process} "
                 f"{volume_flag} "
                 f"--image {sec.image} "
                 f"{paths_start} "
                 f"{cmd} {mpi2} "
                 f"{paths_end} "
             )
-        else:
```

Comment:
Yes, pylint complains about this ( `else-after-return` I think ).    Judging from the name, I think it is pretty fussy about flow-of-control and thinks that if you have a return statement you don't need an else block.   I have a very slight preference to follow along with that b/c in general it can help avoid long nested case blocks, but it is your package, so, up to you.

Reply:
I think I'd prefer to disable this complaint in the config, and allow return-after-else, just for my own clarity.

Reply:
Ok, done.